README.md

```diff
@@ -3,15 +3,17 @@
 ## What is this?
 This explainer outlines the benefits of compression dictionaries, details the different use case for them, and then proposes a way to deliver such dictionaries to browsers to enable these use cases.
 
+The HTTP headers and negotiation are specified in the [IETF Draft document for Compression Dictionary Transport](https://datatracker.ietf.org/doc/draft-ietf-httpbis-compression-dictionary/).
+
 ## Summary
-This proposal adds support for using designated previous responses as an external dictionary for HTTP responses for compression schemes that support external dictionaries (e.g. [Brotli](https://www.rfc-editor.org/rfc/rfc7932) and [Zstandard](https://www.rfc-editor.org/rfc/rfc8878)). 
+This proposal adds support for using designated previous responses as an external dictionary for HTTP responses for compression schemes that support external dictionaries (e.g. [Brotli](https://www.rfc-editor.org/rfc/rfc7932) and [Zstandard](https://www.rfc-editor.org/rfc/rfc8878)).
 
 HTTP `Content-Encoding` is extended with new encoding types and support for allowing responses to be used as dictionaries for future requests. *All actual header values and names still TBD*:
 
-* Server responds to a request for a cacheable resource with an `use-as-dictionary: <options>`.
+* Server responds to a request for a cacheable resource with a `Use-As-Dictionary: <options>` response header.
 * The client will store a hash of the uncompressed response and the applicable `match` URL pattern for the resource with the cached response to identify it as a dictionary.
-* On future requests, the client will match a request against the available dictionary `match` URL patterns. If multiple patterns are matched, the most-specific match is used. If a dictionary is available for a given request, the client will add an appropriate compression scheme (e.g. `sbr` for shared brotli) to the `Accept-Encoding` request header as well as a `sec-available-dictionary: <SHA-256>` header with the hash of the best available dictionary (only SHA-256 is currently supported).
-* If the server has a compressed version of the request URL with the matching dictionary, it serves the dictonary-compressed response with the applicable `Content-Encoding:` (e.g. `sbr`) and `Vary: Accept-Encoding,sec-available-dictionary`.
+* On future requests, the client will match a request against the available dictionary `match` URL patterns. If multiple patterns are matched, the most-specific match is used. If a dictionary is available for a given request, the client will add an appropriate compression scheme (e.g. `br-d` for shared brotli) to the `Accept-Encoding` request header as well as an `Available-Dictionary: <SHA-256>` header with the hash of the best available dictionary.
```

Comment:
Should we say here that SHA-256 is sf-binary, and is a base64 representation in practice?

Reply:
Might also be worthwhile to add a note somewhere indicating that this value is not filesystem safe, and that some "translation" would be required.